src/cascadia/ut_app/JsonTests.cpp

```diff
@@ -87,7 +87,7 @@ namespace TerminalAppUnitTests
                                           "\"purple\" : \"#881798\","
                                           "\"red\" : \"#C50F1F\","
                                           "\"selectionBackground\" : \"#131313\","
-                                          "\"white\" : \"#CCC\","
+                                          "\"white\" : \"#CCCCCC\","

```

Comment:
are we losing a parser test for short color names?

Reply:
Yeah. Adding a test to JsonUtilsTests to cover that case.